src/cleanup.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+import shutil
+
+REMOVE_FILES = ["*.py[co]", "test_infra.log"]
+REMOVE_FOLDERS = ["__pycache__", "UNKNOWN.egg-info", "build", "assisted-service", "reports", ".mypy_cache"]
```

Comment:
I don't think we should remove assisted-service directory, we will have to clone it every time

Reply:
when you do clean, don't you think we should clean everything?

Reply:
We have logic that sets that just resets to master if the repository already exists, so I think i is redundant. I think whenever we can we should avoid cloning

Reply:
changed